docs/sources/datasources/loki/configure-loki-data-source.md

```diff
@@ -0,0 +1,129 @@
+
+The following is a list of configuration options for Loki.
+
+The first option to configure is the name of your connection:
+
+- **Name** - The data source name. This is how you refer to the data source in panels and queries. Examples: loki-1, loki_logs.
+
+- **Default** - Toggle to select as the default name in dashboard panels. When you go to a dashboard panel this will be the default selected data source.
+
+### HTTP section
+
+- **URL** - The URL of your Loki server. Loki uses port 3100. If your Loki server is local, use `http://localhost:3100`. If it is on a server within a network, this is the URL with port where you are running Loki. Example: `http://loki.example.orgname:3100`.
+
+- **Allowed cookies** - Specify cookies by name that should be forwarded to the data source. The Grafana proxy deletes all forwarded cookies by default.
+
+- **Timeout** - The HTTP request timeout. This must be in seconds. There is no default, so this setting is up to you.
+
+### Auth section
+
+There are several authentication methods you can choose in the Authentication section.
+
+{{% admonition type="note" %}}
+Use TLS (Transport Layer Security) for an additional layer of security when working with Loki. For information on setting up TLS encryption with Loki see [Grafana Loki configuration parameters](/docs/loki/latest/configuration/).
+{{% /admonition %}}
+
+- **Basic authentication** - The most common authentication method. Use your `data source` user name and `data source` password to connect.
+
+- **With credentials** - Toggle on to enable credentials such as cookies or auth headers to be sent with cross-site requests.
+
+- **TLS client authentication** - Toggle on to use client authentication. When enabled, add the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server.
+
+- **With CA cert** - Authenticate with a CA certificate. Follow the instructions of the CA (Certificate Authority) to download the certificate file.
+
+- **Skip TLS verify** - Toggle on to bypass TLS certificate validation.
+
+- **Forward OAuth identity** - Forward the OAuth access token (and also the OIDC ID token if available) of the user querying the data source.
+
+### Custom HTTP headers
+
+- **Header** - Add a custom header. This allows custom headers to be passed based on the needs of your Loki instance.
+
+- **Value** - The value of the header.
+
+### Alerting
+
+- **Manage alert rules in Alerting UI** - Toggle on to manage alert rules for the Loki data source. To manage other alerting resources add an `Alertmanager` data source.
+
+### Queries
+
+- **Maximum lines** - Toggle on to set the maximum number of log lines returned by Loki. Increase the limit to have a bigger results set for ad-hoc analysis. Decrease the limit if your browser is sluggish when displaying log results. The default is `1000`.
```

Comment:
This is not a toggleable feature, so I would say something like "Enter a value to...".